nservicebus/upgrades/gateway-2to3.md

```diff
@@ -18,4 +18,9 @@ related:
 
 `NumberOfWorkerThreads` is now deprecated as a parameter for channels in the endpoint config file. Use `MaxConcurrency` to set the maximum number of messages that should be processed at any given time by the gateway instead.
 
-snippet: 1to2GatewayConfig
\ No newline at end of file
+snippet: 1to2GatewayConfig
+
+
+## Notifications
+
+In versions 2 and above the gateway does not expose any error notifications. When an error occurs during sending of a message to other sites, the message will be retried and possibly moved to the error queue but user will not be notified about these events.
```

Comment:
@MarcinHoppe good catch. yeah do the rename here. also dont bother with a redirect since we will eventually want to use 2to3 in the furture